src/main/java/com/musala/atmosphere/agent/util/FtpConnectionManager.java

```diff
@@ -88,6 +84,7 @@ public void disconnect() {
      * @return <code>true</code> if the data transfer is successful, otherwise returns <code>false</code>
      */
     public boolean transferData(File fileToTransfer, String remoteFileName) {
```

Comment:
I think you could remove the second parameter. This method is just transferring the file to the given FTP server. I think for now this is working for us. Leave only the file as a parameter and transfer it with its local name.